common/spec/dependabot/pull_request_creator/github_spec.rb

```diff
@@ -612,6 +612,41 @@
       end
     end
 
+    context "when the branch already exists" do
+      before do
+        service_pack_response.gsub!("heads/rubocop", "heads/#{branch_name}")
```

Comment:
can we use a fixture where a branch exists instead of gsub-ing
the branch name is defined here, so we can change that to match the branch in the fixture https://github.com/dependabot/dependabot-core/blob/acef86558106485491ecf0539b6971878a8ba814/common/spec/dependabot/pull_request_creator/github_spec.rb#L34

Reply:
@Nishnha , added a new fixture to reflect the branch name